p2oc/wallet.py

```diff
@@ -63,3 +69,10 @@ def allocate_funds(amount, lnd, include_tx_fee=True, min_confirmations=6):
         )
 
     return psbt
+
+
+def estimate_fee(lnd, conf_target=6):
```

Comment:
I don't see this being used?

Reply:
yes, it was used in the previous changes i reverted because they were incorrect. I left this method just in case we may need it in the future